src/main.ts

```diff
@@ -37,13 +37,19 @@ const main = async () => {
     ? LocalWorkspace.createOrSelectStack(stackArgs)
     : LocalWorkspace.selectStack(stackArgs));
 
-  core.startGroup(`pulumi ${config.command} on ${config.stackName}`);
-
   const onOutput = (msg: string) => {
     core.debug(msg);
     core.info(msg);
   };
 
+  if (config.refresh) {
```

Comment:
I really like this - separating the refresh out of the log group :)

Reply:
Yeah, thanks!